### PR TITLE
Add notepad++ to the supported editors, on mac and linux it will retu…

### PR DESCRIPTION
**Pull request by jvelezpo**
* Add notepad++ to the supported editors, on mac and linux it will return false since it is not supported
* add notepadpp testing and fix terminal testing

### CODE REVIEW

File: src/editors/notepadpp.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  }
+
+  public async uninstallPlugin(): Promise<void> {
+    throw new Error("Method not implemented.");
+  }
+
+  private appDirectory(): string {
+    switch (os.platform()) {
+      case "win32":
+        return null;
```
Comment (gandarez) [inline src/editors/notepadpp.ts]: returning null?

File: src/editors/notepadpp.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+  public get name(): string {
+    return "Notepadpp";
+  }
+
+  public get icon(): string {
+    return "";
+  }
+
+  public get binaries(): string[] {
```
Comment (gandarez) [inline src/editors/notepadpp.ts]: It should be declared only if you test binaries againt commandExists.exists()

File: src/editors/notepadpp.ts [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  }
+
+  public async uninstallPlugin(): Promise<void> {
+    throw new Error("Method not implemented.");
+  }
+
+  private appDirectory(): string {
+    switch (os.platform()) {
+      case "win32":
+        return "";
```
Comment (gandarez) [inline src/editors/notepadpp.ts]: It's still returning `empty`.

File: src/editors/notepadpp.ts [hunk 4]
```diff
@@ -0,0 +1,60 @@
+import os from "os";
+
+import Editor from "./editor";
+
+export default class Notepadpp extends Editor {
+  public static getName(): string {
+    return "Notepadpp";
```
Comment (alanhamlett) [inline src/editors/notepadpp.ts]: Let's use [Prettier](https://prettier.io/), which will reformat this to single quotes according to our [prettier config](https://github.com/wakatime/desktop/blob/master/.prettier.js).

**Comment by gandarez** [comment]
@jvelezpo is this branch ready for merging?

**Comment by jvelezpo** [comment]
> @jvelezpo is this branch ready for merging?

yes ready to be merge, also tests on `appveyor` are green

**Pull request merged**